cli/azd/test/functional/up_test.go

```diff
@@ -0,0 +1,299 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package cli_test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/azure/azure-dev/cli/azd/pkg/environment/azdcontext"
+	"github.com/azure/azure-dev/cli/azd/pkg/exec"
+	"github.com/azure/azure-dev/cli/azd/test/azdcli"
+	"github.com/joho/godotenv"
+	"github.com/sethvargo/go-retry"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const expectedTestAppResponse = "Hello, `azd`."
+
+func Test_CLI_Up_Down_WebApp(t *testing.T) {
+	t.Skip("azure-dev/834")
```

Comment:
Should this `skip` be here?

Reply:
That's correct. It's existing code being moved out of `cli_test.go`. The deployment is still flaky.